### PR TITLE
Change content creation and editing to the cms item page

### PR DESCRIPTION
**Pull request by Mackiovello**
Makes it possible to create new content on the cms item page. All changes to the content are thus made on the same page.

For #38

### CODE REVIEW

**Comment by Mackiovello** [comment]
Ping @Loudness

**Comment by Mackiovello** [comment]
Hey, @PatrykSzwer! Could you do a quick review on this one as well? I appreciate it!

File: src/Content/ViewModels/CmsItemPage.json.cs [hunk 1]
```diff
@@ -9,5 +10,69 @@ void Handle(Input.SaveTrigger action)
...
+            }
+
+            protected string FirstCharToUpper(string value)
+            {
+                if (string.IsNullOrEmpty(value))
+                {
+                    return value;
+                }
+
+                return value[0].ToString().ToUpper() + value.Substring(1);
```
Comment (PatrykSzwer) [inline src/Content/ViewModels/CmsItemPage.json.cs]: I would change that to `value.First().ToString().ToUpper()...`, I think It's cleaner than magic number, even if obvious one.

**Comment by Mackiovello** [comment]
Fixed what you mentioned in 	1ffe2cb. Thank you for the review! Merging

**Pull request merged**